scripts/lint-staged/eslint.js

```diff
@@ -31,7 +31,13 @@ function groupFilesByPackage() {
   );
 
   for (const file of files) {
-    const packagePath = packagesWithEslint.find(packagePath => file.startsWith(packagePath));
```

Comment:
this wasn't working since convergence started! it was always resolving under v8 `packages/react` package 🚨